price-feeder/config/config.go

```diff
@@ -171,6 +171,14 @@ func (c Config) ProviderPairs() map[provider.Name][]types.CurrencyPair {
 	return providerPairs
 }
 
+func (c Config) ProviderEndpointsMap() map[provider.Name]provider.Endpoint {
```

Comment:
godoc comment is missing

Reply:
https://github.com/umee-network/umee/pull/1791/commits/923039dcdc0e6803d2ddb7bd5b8097df8f541ce9